src/absil/ilread.fs

```diff
@@ -3971,10 +3971,7 @@ let ClosePdbReader pdb =
 #endif
 
 let OpenILModuleReader infile opts = 
-
-#if FX_NO_NATIVE_MEMORY_MAPPED_FILES
-#else
-   try 
+    if not IL.runningOnMono then
```

Comment:
runningOnMono isn't what we want here, because we may be running with Mono on Windows.

I think just check for Windows.